### PR TITLE
sorted raw content file types alphabetically by file type

### PR DESCRIPTION
**Pull request by L928**
sorted raw content file types alphabetically by file type
also added some more file types
i consider this being useful for maintenance

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -3,14 +3,22 @@
...
-*.png filter=lfs diff=lfs merge=lfs -text
+*.bmp filter=lfs diff=lfs merge=lfs -text
+*.exr filter=lfs diff=lfs merge=lfs -text
+*.fbx filter=lfs diff=lfs merge=lfs -text
+*.jpeg filter=lfs diff=lfs merge=lfs -text
+*.jpg filter=lfs diff=lfs merge=lfs -text
+*.mov filter=lfs diff=lfs merge=lfs -text
 *.mp3 filter=lfs diff=lfs merge=lfs -text
+*.mp4 filter=lfs diff=lfs merge=lfs -text
+*.obj filter=lfs diff=lfs merge=lfs -text
```
Comment (MOZGIII) [inline .gitattributes:15]: Not sure about this, might be useful to keep plaintext

Comment (MOZGIII) [reply .gitattributes:15]: Well, checked, diffs are not meaningful; so this is fine!

**Review by MOZGIII — APPROVED** [review]

**Comment by MOZGIII** [comment]
Thanks!

**Comment by L928** [comment]
Thank you for taking care on this :)

**Pull request merged**